### ISSUE TITLE
Training script does not support loss function `plain` though implemented

### ISSUE THREAD

**Issue by eliphatfs** [Thinklab-SJTU/ThinkMatch#34]
See around `train_eval.py` line 282. It does not check whether input is `plain`, though relevant functionality has been implemented at line 106.

**Comment by rogerwwww** [comment]
I am closing this issue which shall be fixed by https://github.com/Thinklab-SJTU/ThinkMatch/commit/aadae4403733c14c9ae3335be00d4995c502132d . If there are still any problems, feel free to reopen this issue.
